app/models/user.rb

```diff
@@ -1,2 +1,12 @@
 class User < ActiveRecord::Base
+  def self.find_or_create_from_auth(auth)
```

Comment:
This is fairly standard practice. If someone has a better, Hound-approved way to do this, please let me know!